### PR TITLE
Update breaking change settings to breaking change policy

### PR DESCRIPTION
**Pull request by gilwong00**
Rename `BreakingChangeSettings` to `BreakingChangePolicy`

### CODE REVIEW

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 1]
```diff
@@ -97,14 +97,14 @@ message CreateMachineUserResponse {
   User user = 1;
 }
 
-message GetBreakingChangeSettingsRequest {}
+message GetBreakingChangePolicyRequest {}
 
-message GetBreakingChangeSettingsResponse {
-  BreakingChangeSettings settings = 1;
+message GetBreakingChangePolicyResponse {
+  BreakingChangePolicy settings = 1;
```
Comment (mfridman) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:103]:
```suggestion
  BreakingChangePolicy policy = 1;
```

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 2]
```diff
@@ -97,14 +97,14 @@ message CreateMachineUserResponse {
...
 }
 
-message UpdateBreakingChangeSettingsRequest {
-  BreakingChangeSettings settings = 1;
+message UpdateBreakingChangePolicyRequest {
+  BreakingChangePolicy settings = 1;
```
Comment (mfridman) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:107]:
```suggestion
  BreakingChangePolicy policy = 1;
```

**Review by mfridman — APPROVED** [review]

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 3]
```diff
@@ -29,9 +29,9 @@ enum BreakingChangeCategory {
   BREAKING_CHANGE_CATEGORY_WIRE_JSON = 2;
 }
 
-message BreakingChangeSettings {
-  bool enabled = 1;
-  bool ignore_unstable_packages = 2;
+message BreakingChangePolicy {
+  optional bool enabled = 1;
```
Comment (bufdev) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:33]: Why was `optional` added here?

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 4]
```diff
@@ -29,9 +29,9 @@ enum BreakingChangeCategory {
...
+  optional bool ignore_unstable_packages = 2;
```
Comment (bufdev) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:34]: Same question.

File: proto/buf/alpha/registry/v1alpha1/admin.proto [hunk 5]
```diff
@@ -29,9 +29,9 @@ enum BreakingChangeCategory {
   BREAKING_CHANGE_CATEGORY_WIRE_JSON = 2;
 }
 
-message BreakingChangeSettings {
-  bool enabled = 1;
-  bool ignore_unstable_packages = 2;
+message BreakingChangePolicy {
```
Comment (bufdev) [inline proto/buf/alpha/registry/v1alpha1/admin.proto:32]: Needs documentation as to what this is.

File: proto/buf/alpha/registry/v1alpha1/admin.proto (hunk 3 above)
Comment (mfridman) [reply proto/buf/alpha/registry/v1alpha1/admin.proto:33]: For partial updates, so we could differentiate between set vs unset bool. The alternative was to use BoolValue, but historically we abused `optional` for this purpose.

**Pull request merged**